edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/BridgeConnection/IMqttBridgeConnector.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
+{
+    using System.Threading.Tasks;
+
+    public interface IMqttBridgeConnector
```

Comment:
Super nit: Can we call these classes/interfaces MqttBroker* instead of MqttBridge*. MqttBridge has a chance to confuse with an actual Mqtt Bridge